docs/snippets/common/storybook-main-fix-imports-autodocs-monorepo.js.mdx

```diff
@@ -0,0 +1,22 @@
+```js
+// .storybook/main.js
+
+export default {
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|ts)'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-interactions',
+  ],
+  framework: {
+    // Replace your-framework with the framework you are using (e.g., react-webpack5, vue3-webpack5)
+    name: '@storybook/your-framework',
+    options: {},
+  },
+  typescript: {
+    reactDocgen: 'react-docgen',
+    skipBabel: true,
+    check: false,
```

Comment:
Are these the relevant properties? Should we highlight that?

Reply:
Yes, they are. I forgot about them. I'll update this to put a bit more emphasis on the required properties